src/ripple/app/tx/impl/NFTokenAcceptOffer.cpp

```diff
@@ -199,19 +199,18 @@ NFTokenAcceptOffer::preclaim(PreclaimContext const& ctx)
             if (auto const dest = so->at(~sfDestination);
                 dest.has_value() && *dest != ctx.tx[sfAccount])
                 return tecNO_PERMISSION;
-        }
-
-        // The account offering to buy must have funds:
-        auto const needed = so->at(sfAmount);
 
-        if (accountHolds(
-                ctx.view,
-                ctx.tx[sfAccount],
-                needed.getCurrency(),
-                needed.getIssuer(),
-                fhZERO_IF_FROZEN,
-                ctx.j) < needed)
-            return tecINSUFFICIENT_FUNDS;
+            // The account offering to buy must have funds:
```

Comment:
I think this fix looks essentially right.  Good job.

However, even though you're only moving braces, this change affects transaction results.  That's kinda the point.  So the change needs to go on an amendment.  I have an example of how that might be done in the top-most commit here:

https://github.com/scottschurr/rippled/commits/gregw-brokered-mode-nft-patch

There may be better ways to do it, so just use that example as a starting place.